roles/ceph-osd/tasks/activate_osds.yml

```diff
@@ -23,8 +23,29 @@
 - include: osd_fragment.yml
   when: crush_location
 
-- name: start and add that the osd service(s) to the init sequence
+- name: start and add that the osd service(s) to the init sequence (before infernalis)
   service:
     name: ceph
     state: started
     enabled: yes
+  when:
+    ansible_distribution == "Ubuntu" or
```

Comment:
I wanted the behavior to stay the same when the role runs with either Ubuntu or an older supported version of ceph/el. Since I assume that the role works on Ubuntu with Infernalis right now (maybe that is a wrong assumption) that's the first check, and the second check should cancel out any ceph that run on el6 (as infernalis is not supported on el6).

Maybe there is a better way, but the or was intentional :)